src/ranges.js

```diff
@@ -252,6 +252,7 @@ const calculateBufferedPercent = function(adjustedRange,
     // overlap duration so that it actually starts at the beginning of referenceRange
     // by including the difference between the two Range's durations
     // This is a work around for the way Flash has no buffer before currentTime
+    // TODO: see if this is still necessary since Flash isn't included
```

Comment:
This entire function can probably just go. There are a few functions in this file that I don't think have been used since simple fetcher logic was implemented

Reply:
We can do this sort of cleanup in another PR though